_episodes/07-find.md

```diff
@@ -514,20 +519,38 @@ about them."
 > wc -l $(find . -name '*.dat') | sort -n
 > ~~~
 > {: .bash}
+>
+> > ## Solution
+> > Find all files with a `.dat` extension in the current directory.
+> > Count the number of lines each of these files contains, then sort the output numerically.
+> {: .solution}
```

Comment:
How about we list (enumerate) them:

1. Find all files with a `.dat` extension in the current directory
2. Count the number of lines in each of the above files
3. Sort the output from step (2) numerically